src/app/tl-alert/tl-alert.component.html

```diff
@@ -0,0 +1,5 @@
+<wvr-alert-element
+[alertType]="alertType"
+[alertClass]="alertClass">
+  <ng-content select="wvr-text, wvr-text-element" ngProjectAs="wvr-text"></ng-content>
```

Comment:
A question we will have to answer is if they are going to want to use custom HTML within the alert component. This will only support text snippets. We may need to make a new issue to add custom html support to the tl-alert.

Reply:
Template for custom alert added - wip

Reply:
Resolved this by removing the closable alert type and added `closable` as an input property for all alert types.